src/components/Chat.tsx

```diff
@@ -0,0 +1,22 @@
+interface Comment {
+  textName: string[];
+  textContent: string[];
+}
+
+function Chat({ textName, textContent }: Comment) {
+  return (
+    <li className="chats">
+      {textName.map((i, a) => {
```

Comment:
```suggestion
      {textName.map((eachComment, index) => {
```

예시로써 위처럼 명시적으로 이름을 짓는 건 어떨까요?